source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -0,0 +1,468 @@
+rules to a port.
+
+They are are indeed a white list - e.g. everything is *denied* by
+default, and the user adds rules that allow traffic. As long as *one*
+rule is matched, the traffic is allowed.
+
+Security group rules can match on L3 and L4 parameters, specifically on
+the protocol (L3), the IP protocol (L4), and destination ports. The remote IP
+address is also a subject for the match conditions, and rules can be
+added to allow *incoming* and *outgoing* traffic.
+
+The API of security group rules can be found [here](https://developer.openstack.org/api-ref/network/v2/#security-group-rules-security-group-rules).
+
+### Activating the feature
+
+Activating the feature means one thing: have the ***deny-all*** security group
+provisioned. 
+
+To do so, two alternatives are possible, and both will result in the group 
+being provisioned:
+
+1. Add a configuration flag to the ovirt-provider-ovn configuration file. When 
+the ovirt-provider-ovn service is started, if this property is configured, then
+the **deny-all** security group will be provisioned - if not alredy.
+
+2. In the engine-setup, present a new question to the oVirt administrator, 
+asking if he wants the security group feature activated. An affirmative reply
+will provision the **deny-all** security group.
+
+### User perspective
+
+The user is meant to provision security group(s) and rule(s) through ansible,
+or directly using a REST client.
+
+This greatly reduces the oVirt-engine impact, since there's no need to 
+implement the REST client side of the API, nor implement these changes in the
+GUI.
+
+Throughout the remainder of the document, ansible will be assumed as the client, given its descriptive yaml syntax. The [os_security_group](https://docs.ansible.com/ansible/2.5/modules/os_security_group_module.html) and [os_security_group_rules](https://docs.ansible.com/ansible/2.5/modules/os_security_group_rule_module.html) will be leveraged for this.
+
+### Mandatory group creation
+
+There are 2 types of required security groups:
+
+1. the **deny-all** security group, which - depending the configuration is created
+at service startup, or during its initial configuration
+
+2. the **allow-dhcp** security group. Creating a subnet on top of an external network will trigger the provision of an **allow-dhcp** security group **for that particular network**.
+
+The ansible representation of these groups is:
+~~~~~
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: deny-all
+    description: security group to drop all IP traffic
+
+- os_security_group:
+    cloud: ovirt
+    state: present
+    name: allow-dhcp-<subnet_uuid>
+    description: security group to allow dhcp-traffic
+~~~~~
+
+***NOTE:*** the **deny-all** security group can be provisioned through ansible,
+during the engine-setup stage. the **allow-dhcp** groups will be created as 
+part of the subnet creation workflow.
+
+### Initial rule creation
```

Comment:
i'm considering adding plantuml diagrams for this flow. 

do you think the text alone is easy to understand?